pkgs/by-name/si/sillytavern/package.nix

```
@@ -0,0 +1,46 @@
+{
+  makeWrapper,
```

```suggestion
  makeBinaryWrapper,
```

makeBinaryWrapper is a little bit faster AFAIK, but it doesn't support shell expansions yet. https://github.com/NixOS/nixpkgs/issues/172583, we are not using shell expansions here so it should be fine.